src/EnergyPlus/OutputProcessor.cc

```diff
@@ -1672,6 +1672,14 @@ namespace OutputProcessor {
 		cCurrentModuleObject = "Meter:Custom";
 		NumCustomMeters = GetNumObjectsFound( cCurrentModuleObject );
 
+        //make list of names for all Meter:Custom since they cannot refer to other Meter:Custom's
+		std::vector<std::string> namesOfMeterCustom;
```

Comment:
@mbadams5 thanks for the suggestions. I will go ahead and implement them but one thing you should know is that I have never seen more than just 3 of these objects in an input file.